desktop/apps/fair/test/routes.coffee

```diff
@@ -261,6 +262,47 @@ describe '#fetchFairByOrganizerYear', ->
     Backbone.sync.args[0][2].success @fairs
     @next.called.should.be.ok()
 
+describe '#fetchFairByOrganizerYear with redirect', ->
+
+  beforeEach ->
+    sinon.stub Backbone, 'sync'
+
+    profile = new Profile _.extend fabricate('fair_organizer_profile'), owner:fabricate('fair_organizer')
+
+    today = moment().utc()
+
+    @fairs = [
+      fabricate('fair', start_at: new Date("2-2-2014"), id: 'the-armory-show-2014', default_profile_id: 'the-armory-show-2014'),
+      fabricate('fair', start_at: new Date("2-2-2015"), id: 'the-armory-show-2015', default_profile_id: 'the-armory-show-2015'),
+      fabricate('fair', start_at: new Date("2-2-2016"), id: 'the-armory-show-2016', default_profile_id: 'the-armory-show-2016')
+      fabricate('fair', autopublish_artworks_at: today.format(), start_at: today.add(7, 'days'), end_at: today.add(14, 'days'), id: "the-armory-show-#{today.year()}", default_profile_id: "the-armory-show-#{today.year()}")
+    ]
+
+    @req = { params: { id: "the-armory-show-#{today.year()}"} }
```

Comment:
What's this for? Looks like you override it in each spec anyway. It used `today` which caught my eye as something that might start to fail next year or something :)